dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -119,80 +102,43 @@ export const makeWindowGuardian = (
 			reportError: (error: Error, feature: string) => void;
 		};
 	};
-	GAData: GADataType;
+	GAData?: GADataType;
 } => {
 	return {
-		config: makeWindowGuardianConfig(CAPIArticle),
-		polyfilled: false,
-		adBlockers: {
-			active: undefined,
-			onDetect: [],
-		},
-		modules: {
-			sentry: {
-				reportError: () => null,
+		config: {
+			// This indicates to the client side code that we are running a dotcom-rendering rendered page.
+			isDotcomRendering: true,
+			isDev: process.env.NODE_ENV !== 'production',
+			stage: stage,
+			frontendAssetsFullURL: frontendAssetsFullURL,
```

Comment:
How do we feel about shorthand properties? I find them easier to parse, especially with longer properties such as this one, and it helps keep the same variable names accross.

We are using it extensively in the `page` object below, with `sentryPublicApiKey`, `dfpAccountId`, `ajaxUrl`, etc.

```suggestion
			frontendAssetsFullURL,
```

Reply:
There’s an [ESLint rule for this](https://eslint.org/docs/latest/rules/object-shorthand.html), so will raise a PR and we can take that discussion there.

Reply:
Defo. I missed this and would have appreciated a nudge from the tooling!